database/console/stubs.go

```diff
@@ -0,0 +1,18 @@
+package console
+
+type Stubs struct {
+}
+
+func (r Stubs) Model() string {
+	return `package models
+
+	import (
```

Comment:
Chore: We should remove the space, the same as below.
```suggestion
import (
```